src/page.js

```diff
@@ -43,17 +43,75 @@ export default class Page {
     off(event) {
         return this.phantom.off(event, this.target);
     }
+    
+    /**
+     * Invokes an asynchronous method
+     */
+    invokeAsyncMethod() {
+        return this.phantom.execute(this.target, 'invokeAsyncMethod', [].slice.call(arguments));
+    }
+    
+    /**
+     * Invokes a method
+     */
+    invokeMethod() {
+        return this.phantom.execute(this.target, 'invokeMethod', [].slice.call(arguments));
+    }
+    
+    /**
+     * Defines a method
+     */
+    defineMethod(name, implementation) {
+        return this.phantom.execute(this.target, 'defineMethod', [name, implementation]);
+    }
+    
+    /**
+     * Gets or sets a property
+     */
+    property() {
```

Comment:
I am trying to figure out why `property` and `setting` got added. Can't they just be part of methods array?